mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1977,7 +2047,7 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 		}
 
 		if statusBeforeUpdate != "TERMINATED" {
```

Comment:
```suggestion
		if statusBeforeUpdate != "TERMINATED" && d.Get("stopping_with_local_ssd_discard").(bool) {
```
I know that it's highly possible that checking `stopping_with_local_ssd_discard` is redundant here, but just want to be extra cautious in case we miss anything in the previous check.

Reply:
added